sites/ol-author-similarity/index.html

```diff
@@ -10,6 +10,7 @@
     <button type="button" id="searchBtn">Go</button>
     <button type="button" id="prevBtn">Prev</button>
     <button type="button" id="nextBtn">Next</button>
+    <div id="author-link"></div>
```

Comment:
```suggestion
    <span id="author-link"></span>
```

Lets make this a span so it shows up on the same line!